bedrock/base/templates/macros.html

```diff
@@ -377,6 +377,16 @@
 </header>{#-- /#fxosnav-header --#}
 {%- endmacro %}
 
+{% macro google_play_button(alt_href='', class_name='', extra_attributes=None, id='', target_blank=False) -%}
```

Comment:
`target_blank` is for `a`, while `extra_attributes` is for `img` inside it. Is it okay to mix them up?